src/zed_rope.ml

```diff
@@ -722,9 +723,16 @@ module Zip_raw = struct
     zip : rope_zipper;
   }
 
+  (* let rec pp_rope t=
+    match t with
+    | Leaf (s,_)-> s
+    | Node (_,_,l,_,r)-> Zed_string.append (pp_rope l) (pp_rope r) *)
+
```

Comment:
If you don't need this code, you can remove it

Reply:
Yup :)